templates/components/carousel.html

```diff
@@ -1,16 +1,22 @@
 <section class="heroCarousel"
     data-slick='{
-        "arrows": {{arrows}},
         "mobileFirst": true,
         "slidesToShow": 1,
         "slidesToScroll": 1,
         "autoplay": true,
         "autoplaySpeed": {{carousel.swap_frequency}},
         "lazyLoad": "anticipated",
-        "accessibility": false
+        "slide": ".hero-slide",
+        "prevArrow": ".js-hero-prev-arrow",
+        "nextArrow": ".js-hero-next-arrow"
     }'>
+    {{#if arrows}}
+    {{#if carousel.slides.length '>' 1}}
```

Comment:
Can we concatenate two conditions?

Reply:
I would like to, but do we have a way to do it in Handlebars?

Reply:
@yurytut1993  should be something like {{#and arrows (if carousel.slides.length '>' 1)}}